ethereum/contracts/IncentivizedInboundChannel.sol

```diff
@@ -90,9 +64,7 @@ contract IncentivizedInboundChannel is InboundChannel {
             // Deliver the message to the target
             // Delivery will have fixed maximum gas allowed for the target app
             (bool success, ) =
-                _messages[i].target.call{value: 0, gas: MAX_GAS_PER_MESSAGE}(
-                    _messages[i].payload
-                );
```

Comment:
Likewise, I think only the checks above with the `require` statements were unnecessary, right?

Reply:
Nevermind, has been added here again: https://github.com/Snowfork/polkadot-ethereum/pull/279